entrypoint.sh

```diff
@@ -1,8 +1,8 @@
 #!/usr/bin/env bash
 
-set -euxo pipefail
+set -eo pipefail
 
-if [ -n $DOCKER_USERNAME ] && [ -n $DOCKER_PASSWORD ]; then
+if [ -z ${DOCKER_USERNAME+x} ] && [ -z ${DOCKER_PASSWORD+x} ]; then
```

Comment:
`-n` returns true if the value is **set**. `-z` returns true if the value is **unset** - which means that we are inverting the if statement. I don't think this is what you want, because we do want to log into docker if those two env vars are set.